### PR TITLE
fix: inlining code examples in upper jaw

### LINKED ISSUE #56035: Building Cat Photo App Issue
### Describe the Issue

So when following the step and things the examples on laptop are not drawing in correctly. They are drawing in, in a way that makes it look like it is 3 lines rather then one line.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-12

### Your code

```
<p>See more <a 
href="https://freecatphotoapp.com">cat photos</a> 
in our gallery.</p>
```

### Expected behavior

I expected it to be right as it looked similar to the example above.

### Screenshots

image

### System

- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Most recent version

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Sembauke**
Examples should not inline as they may cause confusion for the camper. See below mentioned issue.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Before:
image

After:
image

Closes #56035

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
Hey @Sembauke, Huyen's just added a PrismFormatted component to the ui components: https://github.com/freeCodeCamp/ui/pull/303. Could you port this fix in that repo instead?

**Comment by Sembauke** [comment]
Alright, I'll take a look at it!

**Comment by Sembauke** [comment]
I am not entirely sure if it has something to do with the Prism Component itself or with the details element applying it's max width to the child elements. Causing the Prism component to inline instead of using it's content width.

**Comment by ojeytonwilliams** [comment]
I haven't looked into that deeply, but all of this going to be handled by the ui-components, so I assume this fix will need to be there too.

**Comment by Sembauke** [comment]
@huyenltnguyen probably know if the detail element will be part of the component :)

**Review by huyenltnguyen — APPROVED** [review]
I can't reproduce the issue so I can't test the fix. Though I randomly checked some challenges and the challenge description seems to be displaying correctly still.

I can also confirm that the issue isn't in PrismFormatted as I can't reproduce it in fcc/ui (https://github.com/freeCodeCamp/ui/pull/319). I guess we could keep the changes here in /learn, then revisit when we get to a code example/code snippet component (I reckon we'd probably need to build one in fcc/ui in order to share it between /learn and CAP).

File: client/src/templates/Challenges/classic/editor.css [hunk 1]
```diff
@@ -269,6 +269,11 @@ textarea.inputarea {
   cursor: pointer;
 }
 
+.code-details-overflow-container {
+  max-width: 100%;
+  overflow-x: auto;
+}
+
```
Comment (ojeytonwilliams) [inline client/src/templates/Challenges/classic/editor.css:276]:
```suggestion
.editor-upper-jaw pre code {
  white-space: pre;
}

```
This css change should be enough.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
It looks like the bug was introduced here: https://github.com/freeCodeCamp/freeCodeCamp/pull/55681, so it should be enough to provide a more specific selector to fix it:

File: client/src/templates/Challenges/classic/editor.css (hunk 1 above)
Comment (ojeytonwilliams) [reply client/src/templates/Challenges/classic/editor.css:276]: The rest of the changes can be reverted.

**Review by ojeytonwilliams — APPROVED** [review]

**Pull request merged**